src/ceph_osd.cc

```diff
@@ -614,12 +614,23 @@ int main(int argc, const char **argv)
 
   ms_objecter->set_default_policy(Messenger::Policy::lossy_client(CEPH_FEATURE_OSDREPLYMUX));
 
-  entity_addrvec_t public_addrs, cluster_addrs;
+  entity_addrvec_t public_addrs, public_bind_addrs, cluster_addrs;
   r = pick_addresses(g_ceph_context, CEPH_PICK_ADDRESS_PUBLIC, &public_addrs,
 		     iface_preferred_numa_node);
   if (r < 0) {
     derr << "Failed to pick public address." << dendl;
     forker.exit(1);
+  } else {
+    dout(10) << "picked public addrs " << public_addrs << dendl;
```

Comment:
Want to add a corresponding dout for `public_bind_addrs` (currently there is one only for the case when it's defaulted to `public_addrs`)?

Reply:
Sure!